src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -111,7 +113,8 @@ public void verifyImagesInWhatLinksHerePage() {
     links.verifyInfoboxArticleInList(articleName);
   }
 
-  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_004"})
+  // TODO: compress this test
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)